src/common/contexts/TokenInformationContext/TokenInformationContext.tsx

```diff
@@ -17,6 +17,7 @@ interface TokenInformationContext {
   endorseBeneficiary: TitleEscrow["endorseBeneficiary"];
   endorseBeneficiaryState: ContractFunctionState;
   initialize: (tokenRegistryAddress: string, tokenId: string) => void;
+  isEBLSurrendered: boolean;
```

Comment:
It's not always a EBL

```suggestion
  isSurrendered: boolean;
```